### PR TITLE
Added padding and general styling fixes

### PR DESCRIPTION
**Pull request by chrislombaard**

### CODE REVIEW

File: src/themes/default/_containers.scss [hunk 1]
```diff
@@ -10,7 +10,7 @@
         flex-grow: 1;
         height: 90%;
         margin: 0;
-        padding: 0 $rhythm * 2;
+        padding: 10px $rhythm * 2 10px;
```
Comment (jwfwessels) [inline src/themes/default/_containers.scss]: this could just be `10px $rhythm * 2`
padding has 4 argument styles 
https://developer.mozilla.org/en-US/docs/Web/CSS/padding
if you want top and bottom and left and right to be the same you just need 2 values. `<top/botom> <right/left>`

also why 10px? where are you getting that value from?

File: src/themes/default/components/Carousel/_carousel.scss [hunk 2]
```diff
@@ -19,8 +19,9 @@
     }
 
     .Text {
-        border-radius: 0;
+        border-radius: 10px;
         width: 100%;
+        height: 110px;
```
Comment (jwfwessels) [inline src/themes/default/components/Carousel/_carousel.scss]: hm why do you need height? carousels grow based on there contents if i recall?

File: src/themes/default/components/Input/_input.scss [hunk 3]
```diff
@@ -11,7 +11,7 @@
         outline: 0 none;
         background-color: transparent;
         color: $grey-dark;
-        font-size: rem(20);
+        font-size: 16px;
```
Comment (jwfwessels) [inline src/themes/default/components/Input/_input.scss]: please use the rem mixin. `rem(20)`

File: src/themes/default/_containers.scss (hunk 1 above)
Comment (chrislombaard) [reply src/themes/default/_containers.scss]: Ok, I didn't know that you didn't have to repeat padding for top and bottom if their both the same. Noted. It was a guesstimation on what looked good in my opinion.

File: src/themes/default/components/Carousel/_carousel.scss (hunk 2 above)
Comment (chrislombaard) [reply src/themes/default/components/Carousel/_carousel.scss]: Belinda wants a fixed height on these, but let me double check on that.

File: src/themes/default/components/Input/_input.scss (hunk 3 above)
Comment (chrislombaard) [reply src/themes/default/components/Input/_input.scss]: `rem(20)` was failing when I tested on my local machine. Not sure if the function hasn't been imported or if it doesn't exist etc

File: src/themes/default/components/Carousel/_carousel.scss [hunk 4]
```diff
@@ -52,16 +51,14 @@
             @extend .slick-prev;
             right: $message-spacing-thick * -1;
             left: auto;
-            border-top-right-radius: 0px;
-            border-bottom-right-radius: 0px;
-            border-top-left-radius: $message-border-radius-small;
-            border-bottom-left-radius: $message-border-radius-small;
+            border-radius: $message-border-radius-small 0px 0px $message-border-radius-small;
```
Comment (jwfwessels) [inline src/themes/default/components/Carousel/_carousel.scss]: why did you change this?
also why did you convert it to the terse form? it was easier to read previously.
